package/MDAnalysis/coordinates/ParmEd.py

```diff
@@ -173,9 +173,10 @@ def convert(self, obj):
             ag_or_ts = obj.atoms
         except AttributeError:
             if isinstance(obj, base.Timestep):
-                ag_or_ts = obj.copy()
+                raise ValueError("Writing Timesteps to ParmEd "
```

Comment:
Since passing timesteps never really worked, and was removed in 1.1, is the logic here really required (i.e. do we really need to specifically call out passing a timestep)?

Reply:
Probably not, I just copied over the changes from 1.1